fastlane/Fastfile

```diff
@@ -41,4 +41,21 @@ platform :android do
             check_superseded_tracks:    'false',
         )
     end
-end
+	
+	desc "upload Paintroid screenshots"
+	lane :upload_Screenshots_Paintroid do
+		upload_to_play_store(
+		  package_name:               'org.catrobat.paintroid.test',
```

Comment:
```suggestion
		  package_name:               'org.catrobat.paintroid',
```
This determines to which app on the play store they are uploaded. Here we want it for the main app.

Reply:
Changed